src/workers/autounexile.py

```diff
@@ -0,0 +1,24 @@
+from discord.ext import tasks
+from discord.ext.commands import Bot
+from database import exiles_database
+from services.exile_service import unexile_user
+from settings import get_settings
+from .helper import create_autounexile_embed
+
+settings = get_settings()
+
+
+@tasks.loop(minutes=1.0)
+async def autounexile_users(self):
+    exiles = exiles_database.get_pending_unexiles()
+
+    for exile in exiles:
+        member = self.get_guild(settings.guild_id).get_member(exile.user_id)
+        if member is not None:
```

Comment:
we should do some error logging in case the member cant be found. let's do this only to the logging buffer and not the embed